python/dask_cudf/dask_cudf/expr/__init__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+from dask import config
+
+# Check if dask-dataframe is using dask-expr
+QUERY_PLANNING_ON = config.get("dataframe.query-planning", False)
+
+# Register custom expressions and collections
+try:
+    import dask_cudf.expr._collection
+    import dask_cudf.expr._expr
+
+    DASK_EXPR_ENABLED = True  # Dask-expr is installed
```

Comment:
This name means something different than the `DASK_EXPR_ENABLED` variable in `dask.dataframe`. Should probably be removed from `dask_cudf`, since it isn't used anyway.